Source/Core/VideoCommon/FrameDumpFFMpeg.cpp

```diff
@@ -194,7 +193,7 @@ bool FFMpegFrameDump::PrepareEncoding(int w, int h, u64 start_ticks, u32 savesta
 
 bool FFMpegFrameDump::CreateVideoFile()
 {
-  const std::string& format = g_Config.sDumpFormat;
+  const std::string& format = Config::Get(Config::GFX_DUMP_FORMAT);
```

Comment:
Using a const reference for a value returned by copy is unnecessarily confusing IMO. (As I understand it, it isn't UB, but there's no advantage to it compared to making this variable `std::string format`.)

Reply:
I've made them just `const std::string` now.